app/src/cc/arduino/view/preferences/Preferences.java

```diff
@@ -826,7 +831,7 @@ private void savePreferencesData() {
 
     PreferencesData.setBoolean("update.check", checkUpdatesBox.isSelected());
 
-    PreferencesData.setBoolean("editor.save_on_verify", saveVerifyUploadBox.isSelected());
```

Comment:
I think this line should be preserved; it's probably a copy-paste error

Reply:
I removed that line because it was identical to the line 2 above it, and it didn't seem to need to be there twice.

Reply:
Ah great, I didn't catch it. Would you mind creating a separate commit the removes that line so the history is cleaner?